### DISCUSSION TITLE
How to redirect output to /dev/null

### DISCUSSION THREAD

**Discussion by MooreMachine** [community/community#26616] [Actions]
<p>I’m setting up a composite action; at some point I compile and run an application to make sure it completes its execution without throwing an error. Its output is irrelevant however (I just want to make sure that the executable returns 0). How can I redirect the output to <code>/dev/null</code> so that it doesn’t clutter the console/log?</p>
<p>In a normal Linux environment I would do <code>./myapp 1&gt; /dev/null</code> but that doesn’t seem to work with the GitHub-hosted runners.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p><code>1&gt;</code> redirects only stdout, if there is any stderr output you don’t want to see you’ll need to redirect that, too.</p>
<p>If that doesn’t help, which shell is your step using? Could you show an example of the problem?</p>

**Comment by MooreMachine** [comment]
<p>Thank you airtower-luna :slight_smile: I do want to see the stderr output, that’s why I only want to redirect stdout. But I just went back to check my code and you’re right, there were lines that were outputting to stderr (which I’m not redirecting).</p>
<p>So my bad! There’s no problem with redirection while running on a hosted runner.</p>
<p>Thank you again!</p>
